cli/cmd/version.go

```diff
@@ -12,28 +12,43 @@ import (
 
 const DefaultVersionString = "unavailable"
 
+var shortVersion bool
+var onlyClientVersion bool
+
 var versionCmd = &cobra.Command{
 	Use:   "version",
 	Short: "Print the client and server version information",
-	Long:  "Print the client and server version information.",
-	Args:  cobra.NoArgs,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Printf("Client version: %s\n", version.Version)
+		clientVersion := version.Version
+		if shortVersion {
+			fmt.Println(clientVersion)
+		} else {
+			fmt.Printf("Client version: %s\n", clientVersion)
+		}
 
-		client, err := newPublicAPIClient()
+		conduitApiClient, err := newPublicAPIClient()
 		if err != nil {
 			fmt.Fprintf(os.Stderr, "Error connecting to server: %s\n", err)
 			os.Exit(1)
 		}
 
-		fmt.Printf("Server version: %s\n", getServerVersion(client))
+		if !onlyClientVersion {
+			serverVersion := getServerVersion(conduitApiClient)
+			if shortVersion {
+				fmt.Println(serverVersion)
+			} else {
+				fmt.Printf("Server version: %s\n", serverVersion)
+			}
+		}
 
 		return
 	},
 }
 
 func init() {
 	RootCmd.AddCommand(versionCmd)
+	versionCmd.PersistentFlags().BoolVar(&shortVersion, "short", false, "Display only the version number(s), with no additional formatting.")
+	versionCmd.PersistentFlags().BoolVar(&onlyClientVersion, "client", false, "Client version only.")
```

Comment:
Should we have a `--server` flag for completeness?  Although how would they interact? (setting both flags is invalid?)

Reply:
Yeah, I thought about that but ultimately followed the `kubectl` precedent:

```shell
pcalcado@arbeitsziege.local $ kubectl help version
Print the client and server version information for the current context

Examples:
  # Print the client and server versions for the current context
  kubectl version

Options:
  -c, --client=false: Client version only (no server required).
  -o, --output='': One of 'yaml' or 'json'.
      --short=false: Print just the version number.

Usage:
  kubectl version [options]

Use "kubectl options" for a list of global command-line options (applies to all commands).
```